game/src/Game/GameSession/MapView.gd

```diff
@@ -20,15 +20,18 @@ var _drag_anchor : Vector2
 var _drag_active : bool = false
 
 var _mouse_over_viewport : bool = true
+var _window_in_focus : bool = true
 
 @export var _zoom_target_min : float = 0.05
 @export var _zoom_target_max : float = 5.0
 @export var _zoom_target_step : float = 0.1
-@export var _zoom_epsilon : float = _zoom_target_step * 0.1
+@export var _zoom_epsilon : float = _zoom_target_step * 0.005
```

Comment:
A smaller `_zoom_epsilon` makes zooming feel much smoother, especially when close to the map. Based on my testing this value feels right.